static/app/components/organizations/pageFilterRow.tsx

```diff
@@ -32,38 +33,62 @@ function PageFilterRow({
   renderCheckbox = defaultRenderCheckbox,
   ...props
 }: Props) {
-  const checkbox = <CheckboxFancy isDisabled={!multi} isChecked={checked} />;
+  const rowId = useRef(domId('page_filter_row'));
+
+  const checkbox = (
+    <MultiselectCheckbox
+      isDisabled={!multi}
+      isChecked={checked}
+      onClick={multi ? onCheckClick : undefined}
+      aria-labelledby={rowId.current}
+    />
+  );
 
   return (
-    <Container isChecked={checked} {...props}>
-      <Content multi={multi}>{children}</Content>
-      <CheckboxHitbox onClick={multi ? onCheckClick : undefined}>
```

Comment:
This hitbox thing wasn't great since it had poor accessibility. We can replace it by styling the checkbox and adding a pseudo element behind it to expand the hitbox.